cross/lua/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = lua
+PKG_VERS = 5.3.4
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://www.lua.org/ftp/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+BUILD_DEPENDS = cross/ncurses cross/readline
```

Comment:
Please move these to `DEPENDS`, so that there is no need for copy files to lua `PLIST`